lib/src/test/kotlin/starknet/utils/DevnetClient.kt

```diff
@@ -51,12 +55,67 @@ class DevnetClient(val host: String = "localhost", val port: Int = 5050) {
         deployProcess.waitFor()
 
         val result = String(deployProcess.inputStream.readAllBytes())
+        val lines = result.lines()
+        return getTransactionResult(lines)
+    }
+
+    fun declareContract(contractPath: Path): TransactionResult {
```

Comment:
Do you think it's better to take whole Path as a parameter or just a name, and add path inside this method?

Reply:
I think it'd be better to take a whole path: It makes the whole method be more flexible. I guess `deployContract` could be changed to be uniform with `declareContract`. What do you think?

Reply:
Yeah, we can do it this way